test/rails/controller_test.rb

```diff
@@ -233,6 +233,15 @@ class ActiveRecordPresentTest < ActionController::TestCase
 
     assert_equal "active_record_bands/show.html: Band, Band, true, Band::Update", response.body
   end
+
+  test "#collection" do
+    Band.destroy_all
+    Band::Create[band: {name: "Nofx"}]
+
+    get :index
+
+    assert_equal "active_record_bands/index.html: Band::ActiveRecord_Relation, Band::ActiveRecord_Relation, Band::Index", response.body
```

Comment:
there was the [ActiveRecordBandsController#index](https://github.com/apotonick/trailblazer/blob/master/test/rails/fake_app/controllers.rb#L123) but wasn't being called in a test, this line I sent to you, uses the `@bands` in the controller